bahmni-pacs/build.gradle

```diff
@@ -62,21 +62,21 @@ task extractJBoss(dependsOn: ["downloadJBoss"], type: Copy) {
 }
 
 task downloadOviyam2(type: Download) {
-    src 'http://repo.mybahmni.org.s3.amazonaws.com/packages/build/Oviyam-2.1-bin.zip'
+    src 'https://webwerks.dl.sourceforge.net/project/dcm4che/Oviyam/2.8.1/Oviyam-2.8.1-bin.zip'
```

Comment:
@angshu would this require uploading to our bahmni repo ?

Reply:
yess. for now, I feel its ok. We need to upload to our repo with the zip